test/e2e/logexplorationapi/api/deploy_and_collect_logs_from_elasticsearch_test.go

```diff
@@ -0,0 +1,66 @@
+package api
+
+import (
+	"fmt"
+	apps "k8s.io/api/apps/v1"
+	"runtime"
+
+	"github.com/ViaQ/logerr/log"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/cluster-logging-operator/test/helpers"
+)
+
+var _ = Describe("Deploys API and collect logs from elasticsearch", func() {
+	_, filename, _, _ := runtime.Caller(0)
+	log.Info("Running ", "filename", filename)
+	var (
+		e2e                         = helpers.NewE2ETestFramework()
+		components                  []helpers.LogComponentType
+		err                         error
+		logExplorationAPiDeployment *apps.Deployment
+	)
+	Describe("Deploying API and collecting logs from elasticsearch", func() {
```

Comment:
same here

Reply:
Replaced "query" with "collect" in pull request #1208.